From 052cbec90c58bb0126e38de19aeb9fae789b2349 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Tue, 17 Apr 2012 20:22:24 -0300
Subject: [PATCH 33/40] 4.9.0 Valle Sombraluna: Aliado del Ala Abisal

Disabled quest 10871 for players, it's a test version of quest 10870 never released
---
 .../reanemu/2012_04_17_100_world_disables.sql      |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_17_100_world_disables.sql

diff --git a/sql/updates/world/reanemu/2012_04_17_100_world_disables.sql b/sql/updates/world/reanemu/2012_04_17_100_world_disables.sql
new file mode 100644
index 0000000..cfe51de
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_17_100_world_disables.sql
@@ -0,0 +1,4 @@
+-- Ally of the Netherwing, Disabled one of them
+DELETE FROM `disables` WHERE `SourceType`=1 AND `entry`=10871;
+INSERT INTO `disables` (`SourceType`,`entry`,`flags`,`params_0`,`params_1`,`comment`) VALUES
+(1,10871,0,0,0,'Ally of the Netherwing - test version disabled for players');
\ No newline at end of file
-- 
1.7.9.msysgit.0

